packages/components/message-dialog/src/message-dialog.scss

```diff
@@ -0,0 +1,7 @@
+sl-dialog {
+  --sl-dialog-max-inline-size: 20rem;
+}
+
+p {
+  margin-block: 0;
```

Comment:
I think we need to have for the content a variable to set the font size like `--sl-dialog-body-font-size` or sth to make it possible to override by the developer.

Reply:
In this specific case, I just want to make sure the paragraph doesn't have any margins. If I remove this rule, then the chances that outside styling "breaks" the message dialog is big. Also the more I think about it, the more I think the dialog should just use the `<body>` font.

Reply:
Ok, so let's add `font-size: 1.4rem` to the `body` in `base.sccs` file:

<img width="524" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/eaa4cd26-8c1b-4780-9926-9fc64615e318">